src/Components/App.js

```diff
@@ -14,11 +15,18 @@ class App extends Component {
 
   getPlayerStatus() {
     return {
-      'intro': <Intro />,
-      'room': <Room/>
+      'intro': <Intro render={this.renderArea}/>,
+      'room': <Room render={this.renderArea}/>,
+      'map': <Map render={this.renderArea}/>
     }[this.state.playerStatus]
   }
 
+  renderArea = (string) => {
```

Comment:
We should use the getPlayerStatus or your renderArea, not both. I think we should use the renderArea and adjust our code. I can make these adjustments but fyi.